docs/src/man/comparisons.md

```diff
@@ -235,6 +235,59 @@ The table below compares more advanced commands:
 | DataFrame as input        | `summarize(df, head(across(), 2))`                        | `combine(d -> first(d, 2), df)`                               |
 | DataFrame as output       | `summarize(df, tibble(value = c(min(x), max(x))))`        | `combine(:x => x -> (value = [minimum(x), maximum(x)],), df)` |
 
+## Comparison with the R package data.table
+
+The following table compares the main functions of DataFrames.jl with the R package data.table (version 1.14.1). To contrast data.table with dplyr, the focus has been put on transformations in place.
+
+|Operations              | data.table  | DataFrames.jl|  
+|:-----------------------|:------------|:-------------|
+|Reduce multiple values  |`df[, list(mean(x))]`       |`combine(df, :x => mean)`|
```

Comment:
this is not the same as the resulting variable name does not match.

Reply:
I think they are conceptually the same. data.table automatically names variable V1, V2, ... and dataframes appends the function name. 
I dont think anyone would ever write: `df[, list(x_mean = mean(x)) ]` if we just wanted to visualize the mean?

Happy to change it if you think the match has to be exactly the same, but this was made to illustrate a case where you create a new variable and let the framework pick the name.

Reply:
OK - but the difference is that `V1` is a meaningless name and I would never want such a name in my data frame :).
But this is probably a matter of style, so I am OK to leave it as is.

Reply:
Take it to Matt Dowle ;)

Reply:
Two minor comments that @eloualiche is almost certainly aware of, but just in case it's useful...

1. Much of a muchness, but most data.table users prefer the more concise `.()` instead of `list()` in my experience. I seem to recall that the former is also the preferred syntax in the vignettes.
2. On that note, you don't need to coerce to `list()` / `.()` for single variable reduction. E.g. `df[, mean(x)]` and df[, mean(x), by = grp] would both work.

I agree that renaming aggregation variables in place is probably a matter of style and, if the main purpose is to help data.table users navigate in DF.jl, then keeping things simple is best. FWIW the idiomatic way of doing this is via `.SD(cols)`, which Erik documents further down in the PR.

Reply:
Thanks Grant. I thought having the dot here might be confusing. As I said this is mostly for people coming from data.table so I erred on the side of being explicit rather than short (though sometimes it's a tradeoff).